node/src/services/blockchain/commands.rs

```diff
@@ -0,0 +1,116 @@
+use anyhow::Result;
+use jsonrpsee::tracing::event;
+use serde_json::Number;
+use sp_core::H256;
+use storage_hub_infra::actor::ActorHandle;
+
+use super::{handler::BlockchainService, types::Extrinsic};
+
+/// Commands that can be sent to the BlockchainService actor.
+#[derive(Debug)]
+pub enum BlockchainServiceCommand {
+    SendExtrinsic {
+        call: storage_hub_runtime::RuntimeCall,
+        callback: tokio::sync::oneshot::Sender<(tokio::sync::mpsc::Receiver<String>, H256)>,
```

Comment:
In simple terms, the Sender sends to the overlaying task, the Receiver that it gets from the RPC call, so that the overlaying task can get updates from the RPC receiver.